### DISCUSSION TITLE
How to get JSON for an input ´Counter_n´ value on Tasmota 9.5?

### DISCUSSION THREAD

**Discussion by jayjupdhig** [arendst/Tasmota#12433] [Q&A] [answered]
On older version, i was able to get it with IP_ADDRESS/ay... but now that seems to be gone.

How to get JSON for an input ´Counter_n´ value on Tasmota 9.5?

THank you very much for your feedbacks.

**Comment by sfromis** [comment, marked as answer]
The primary way is subscribing to the device on MQTT. No commands needed.

You can also get values via commands like Counter and Status 10. Commands can be [issued](https://tasmota.github.io/docs/Commands/#with-mqtt) via MQTT, serial or http.

**Reply by jayjupdhig** [reply]
Thank you for your feedback. I know that, and i will be published correctly to the MQTT topic. But i need to know how to get that value via the HTTP/JSON (REST?) way - no the MQTT one...

To enter http://IP_ADDRESS/ay will give me a 404 now - compared to older versions.

**Reply by barbudor** [reply]
As pointed by sfromis, you can send a command through web request : https://tasmota.github.io/docs/Commands/#with-web-requests

**Reply by sfromis** [reply]
What ever you may have hacked together with that "ay", it has not been an API, and just with no expectation of stability. Use the official commands, but beware that compared to MQTT, http is slower, less reliable, and without the same features.

**Reply by jayjupdhig** [reply]
@barbudor: Thank you very much!!

@sfromis : Yes i know... that is not the API, it was just a plain webseite to view in the browser. I only used it to prototype the (non-supported, Counter-type-based) input of the Eltako wind sensor on an older version 5.8 or 6.5. (I don't remember right which version it exactly was...)
